Mu2eG4/geom/geom_common_no_ds.txt

```diff
@@ -0,0 +1,13 @@
+//
+// Variant on geom_common.txt for production running of G4 jobs
```

Comment:
Is it required that bfgeom files be read as part of geom_common*?  I was imagining they would be standalone flies read separately by the BFManager service, not as part of geometry.